superset-frontend/src/explore/components/ControlHeader.tsx

```diff
@@ -78,12 +82,13 @@ const ControlHeader: FC<ControlHeaderProps> = ({
       >
         {description && (
           <span>
-            <InfoTooltipWithTrigger
```

Comment:
Remove `InfoTooltipWithTrigger` import in this file

Reply:
It's still used on line 96 to render the lightning bolt icon, do you think I should replace that with a plain `<Tooltip>...</Tooltip>` as well?